pkg/liveshare/test/server.go

```diff
@@ -242,16 +242,17 @@ func handleRequests(ctx context.Context, server *Server, channel ssh.Channel, re
 	errc := make(chan error, 1)
 	go func() {
 		for req := range reqs {
-			if req.WantReply {
-				if err := req.Reply(true, nil); err != nil {
+			r := req
```

Comment:
This is an unrelated change but was giving a diagnostics lint warning that we were using `req` in the function call to `forwardStream`. Using the loop variable in function calls like that is a common source of Go bugs.